RealmSwift/List.swift

```diff
@@ -314,14 +314,16 @@ public final class List<T: Object>: ListBase {
 
      - parameter index: The index at which to remove the object.
      */
-    public func remove(objectAtIndex index: Int) {
+    public func remove(at index: Int) {
```

Comment:
It looks like `Array` returns the value that was there, but `Collection`/`RangeReplaceableCollection` don't and there's alternative functions on `Collection` for remove and return old (`popLast()` etc.), so I'm inclined to leave it as a void return.